circleci/catalog-sync

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Results in catalog-sync-service generating and publishing an updated 
+# app config for the given app.
+#
+# Usage:
+#
+#   catalog-sync [URL] [USER] [PASS] [APP_NAME]
```

Comment:
nit: could be more descriptive than `URL` since a newbie would not know this is the URL circle-ci-integrations (I assume?)